packages/playwright/src/common/config.ts

```diff
@@ -223,6 +223,14 @@ function resolveReporters(reporters: Config['reporter'], rootDir: string): Repor
   });
 }
 
+function resolveWorkes(workers: string) {
```

Comment:
nit: typo in function name

```suggestion
function resolveWorkers(workers: string) {
```

Reply:
Thanks! I will replace it!